homeassistant/components/nuki/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow to configure the Nuki integration."""
+
+from pynuki import NukiBridge
+from requests.exceptions import RequestException
+import voluptuous as vol
+
+from homeassistant import config_entries, exceptions
+from homeassistant.const import CONF_HOST, CONF_PORT, CONF_TOKEN
+
+from .const import DEFAULT_PORT, DEFAULT_TIMEOUT, DOMAIN
```

Comment:
You have to add the pylint disable annotation
```suggestion
from .const import DEFAULT_PORT, DEFAULT_TIMEOUT, DOMAIN # pylint: disable=unused-import
```